src/Controller/EnMarche/CitizenProjectController.php

```diff
@@ -45,4 +48,35 @@ public function helpAction(): Response
 
         return new Response();
     }
+
+    /**
+     * @Route("/comite/autocompletion",
+     *     name="app_citizen_project_committee_autocomplete",
+     *     condition="request.isXmlHttpRequest()"
+     * )
+     * @Method("GET")
+     * @Security("is_granted('IS_AUTHENTICATED_FULLY')")
+     */
+    public function committeeAutocompleteAction(Request $request)
+    {
+        if (!$request->query->get('term', false)) {
```

Comment:
No need to pass `false` as second argument. You can keep the result in a variable to reuse it below.

Reply:
✔️